channels/views_test.py

```diff
@@ -568,7 +572,36 @@ def test_update_post_text(client, private_channel_and_contributor, reddit_factor
         'channel_title': channel.title,
         "profile_image": user.profile.image_small,
         "author_name": user.profile.name,
-        'edited': False
+        'edited': False,
+        "stickied": False,
+    }
+
+
+def test_update_post_stickied(client, private_channel_and_contributor, reddit_factories, staff_user):
+    """Test updating just the stickied boolean on a post"""
+    channel, user = private_channel_and_contributor
+    post = reddit_factories.text_post('just a post', user, channel=channel)
+    client.force_login(staff_user)
+    url = reverse('post-detail', kwargs={'post_id': post.id})
+    resp = client.patch(url, format='json', data={"stickied": True})
+    assert resp.status_code == 200
```

Comment:
Could you change this to use `status.HTTP_200_OK` instead of the magic number?

Reply:
Oh sure, I had just copied this from an existing test so I'll change those too

Reply:
:woman_facepalming: I meant to write `serializers.BooleanField`